scripts/install-appstudio-kcp.sh

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+#
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+command -v kubectl >/dev/null 2>&1 || { echo "kubectl is not installed. Aborting."; exit 1; }
+command -v oc >/dev/null 2>&1 || { echo "oc cli is not installed. Aborting."; exit 1; }
+
+# tr is not working by default in MacOs. Exporting LC_CTYPE=C seems like solve the problem
+case "$(uname -s)" in
+    Darwin*)    export LC_CTYPE=C;;
+    Linux*)     echo -e "Running on Linux system";;
+    *)          echo -e "UNKNOWN System."
+esac
+
+export ROOT_E2E="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+export WORKSPACE=${WORKSPACE:-${ROOT_E2E}}
+export MY_GIT_FORK_REMOTE="qe"
+export MY_GITHUB_ORG="redhat-appstudio-qe"
+export MY_GITHUB_TOKEN="$GITHUB_TOKEN"
+export WORKSPACE_ID=$(tr -dc a-z0-9 </dev/urandom | head -c 5 ; echo '')
```

Comment:
Could we have this one configurable as well? I see a use case when the install failed for some reason (for example kcp outage :) and user might want to rerun the unfinished install with the same workspace IDs